jsk_rosbag_tools/tests/test_jsk_rosbag_tools.test

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<launch>
+
+  <test test-name="test_bag_to_video" pkg="jsk_rosbag_tools" type="test_jsk_rosbag_tools.py"
+        time-limit="3600.0" />
```

Comment:
I am afraid this test takes too long time, increasing the total test time of `jsk_common`, could you consider shoulder version?